index.html

```diff
@@ -53,15 +53,13 @@
     </head>
 
     <body onload="newPassage()" style="font-family: fantasy; background-color: #4A80C1;">
+        <button class="buttonStyle" onclick="newPassage()">New Passage</button>
```

Comment:
could you move this beneath the text to it looks nice

Reply:
no